<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Busc_Historial</class>
 <widget class="QDialog" name="Busc_Historial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buscar/modificar historial</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Berlin Sans FB Demi</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Ingrese identificador de paciente&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_buscar_dni">
           <property name="text">
            <string>DNI</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="buscar_dni"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="buscarpushbuton">
         <property name="font">
          <font>
           <family>Berlin Sans FB Demi</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Buscar</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="varon">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Masculino</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mujer">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Femenino</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="modificarpushbuton">
         <property name="font">
          <font>
           <family>Berlin Sans FB Demi</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Modificar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelarpushbuton">
         <property name="font">
          <font>
           <family>Berlin Sans FB Demi</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="nombre"/>
         </item>
         <item>
          <widget class="QLineEdit" name="edad"/>
         </item>
         <item>
          <widget class="QLineEdit" name="fechanacimiento"/>
         </item>
         <item>
          <widget class="QLineEdit" name="peso"/>
         </item>
         <item>
          <widget class="QLineEdit" name="altura"/>
         </item>
         <item>
          <widget class="QLineEdit" name="dni"/>
         </item>
         <item>
          <widget class="QLineEdit" name="direccion"/>
         </item>
         <item>
          <widget class="QLineEdit" name="enfermedades"/>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_nombre">
           <property name="text">
            <string>Nombre Completo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_edad">
           <property name="text">
            <string>Edad</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_fecha">
           <property name="text">
            <string>Fecha de Nacimiento</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_peso">
           <property name="text">
            <string>Peso</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_altura">
           <property name="text">
            <string>Altura</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_dni">
           <property name="text">
            <string>DNI</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_direccion">
           <property name="text">
            <string>Direcci√≥n</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_enfermedades">
           <property name="text">
            <string>Enfermedades Previas</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
